Sourcery/Configuration.swift

```diff
@@ -106,19 +106,22 @@ struct Configuration {
     let templates: Paths
     let output: Path
     let args: [String: NSObject]
+    let autoLock: Bool
 
-    init(dict: [String: Any], relativePath: Path) {
+    init(dict: [String: Any], relativePath: Path, autoLock: Bool = false) {
```

Comment:
I like the feature as well, but maybe it makes sense to have it true by default? I think the case to have not locked should be an edge case or as @ilyapuchka mentioned - the watch mode.